src/schedule/2021-01-26-some-antics-redwoodjs-tutorials.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Some Antics: Putting RedwoodJS Docs to the Test with Anthony Campolo'
+type: Live Event
+date: 2021-01-26T12:00:00-06:00
+speakers:
+  - Ben Myers
+  - Anthony Campolo
+---
+
+{% twitch 'someanticsdev' %}
```

Comment:
Okay, @chantastic - I'm _really_ digging this Twitch shortcode.

Reply:
woohoo! 

looking good.
quick question. does twitch offer other types of embeds as well?

one of the reasons i build the YouTube embed to be `youtube-video` is to disambiguate it from other possible YouTube embeds.

A cursory look at [the docs](https://dev.twitch.tv/docs/embed/video-and-clips/) seem to suggest that `channel`, `video`, and `collection` all use the same url structure.
If that's right, i'm totally on board with a `twitch` shortcode.
But just wanted to verify first.

Reply:
Yeah, I was going off of the [_Embed Everything_](https://dev.twitch.tv/docs/embed/everything) docs. Embedding a channel like these seemed like the most likely use case, which was why I called it `twitch` instead of something longer.

Something worth noting is that VODs expire after 14 days or 60 days, depending on the broadcaster's status, so Twitch VODs won't make a great source of record for this — chances are good we'll have a lot of content (like Some Antics) that start out on Twitch and then get uploaded to YouTube after the fact. So I'm not sure I want to directly encourage VOD embeds.